sql/rpl_constants.h

```diff
@@ -63,4 +67,126 @@ enum ExtraRowInfoFormat {
   ERIF_MULTI        =  255
 };
 
+static const size_t ENCRYPTION_MASTER_KEY_NAME_MAX_LEN = 100;
+static const size_t ENCRYPTION_SERVER_UUID_LEN = 36;
+static const size_t ENCRYPTION_KEY_LEN = 32;
+
+#define BINLOG_CRYPTO_SCHEME_LENGTH 1
+#define BINLOG_KEY_VERSION_LENGTH   4
+#define BINLOG_IV_LENGTH            MY_AES_BLOCK_SIZE
+#define BINLOG_IV_OFFS_LENGTH       4
+#define BINLOG_NONCE_LENGTH         (BINLOG_IV_LENGTH - BINLOG_IV_OFFS_LENGTH)
+
+struct Binlog_crypt_data {
```

Comment:
Consider making it more of a class with encapsulated data, but ATM I don't have a good design suggestion

Reply:
Done